config/galaxy.ini

```diff
@@ -1,1160 +0,0 @@
-#
```

Comment:
Thanks @djcomlab! Can you please revert the commit that changes this file, so that we keep the one with all comments... promise to deal with this soon on Helm side...